static/src/js/components/BoundingRectangleField/BoundingRectangleField.jsx

```diff
@@ -15,9 +16,17 @@ import './BoundingRectangleField.scss'
  * @param {BoundingRectangleField} props
  */
 const BoundingRectangleField = ({
+  name,
   formData,
-  onChange
+  onChange,
+  schema
 }) => {
+  const { properties } = schema
+  const { EastBoundingCoordinate: EBoundingCoordinate } = properties
```

Comment:
Variable names with upper cased first letters typically indicate they are the name of a class, so this should be renamed to `eastBoundingCoordinate`.

Avoid shortening or abbreviating variable names, the more descriptive they are, the easy they are to read and maintain.

You can destructure both coordinates from `properties` in one call


```
const { 
  EastBoundingCoordinate: eastBoundingCoordinate,
  NorthBoundingCoordinate: northBoundingCoordinate
} = properties
```

Reply:
I went with e instead of east because eastBoudningCoordinate is already being used elsewhere, but yes I can change it otherwise.

Reply:
Ah ok, yeah I think eslint will complain about that. I'd probably add `Properties` to the end, or something like that. `eastCoundingCoordinateProperties`